source/installation/galette.rst

```diff
@@ -38,48 +38,14 @@ Next screen ask you for installation type. You'll choose beetween "New installat
 Database
 ========
 
-First of all, make sure the database exists, and prepare all required information:
+First of all, make sure the database exists, and prepare all required information (please refer to :ref:`database creation section <installationcreatedb>`):
```

Comment:
At the very beginning of the file, we have "once preparation steps are done" with a link to preparation page; I guess that's not useful to repeat it here
```suggestion
First of all, make sure the database exists, and prepare all required information:
```

Reply:
If you think another link here make sense; maybe could we just add it on another line, to make that more clear, and to prevent existing translations to be marked as fuzzy  ;)